ci-operator/step-registry/enable-qe-catalogsource/logging/enable-qe-catalogsource-logging-commands.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
```

Comment:
Any reason to define enable-qe-catalogsoruce-logging here?  can we use the step enable-qe-catalogsource-logging directly?

Reply:
@anpingli The existing enable-qe-catalogsource uses a hardcoded index with tag variable which is set accrodin to OCP version. https://github.com/openshift/release/blob/501f89416aea7c5966b631c7f73fb2e43fc656f5/ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh#L122 
With the one we are adding, we can set our own image and tag and make any customization in future if needed. Also if there is any existing qe-app-registry catalogsource on the cluster, our steps will remove it first and add our own catalogsource and ICSP.